src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -376,6 +379,63 @@ public async Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestP
             }
         }
 
+        /// <summary>
+        ///     Parse out the owner and repo from a repository url
+        /// </summary>
+        /// <param name="prBody">Pull Request commit msg</param>
+        /// <returns>Tuple of owner and repo</returns>
+        public static string ParsePullRequestBody(string prBody)
+        {
+            var matches = DependencyUpdatesPattern.Matches(prBody);
+            if (matches.Count == 0)
+            {
+                return prBody;
+            }
+            string message = "";
+            foreach (Match match in matches)
+            {
+                message += $@"
+{match.Value.ToString().Replace("*", "")}";
```

Comment:
why is this removing the "*" characters?

Reply:
This is how the PR description looks like 

This pull request updates the following dependencies

[marker]: <> (Begin:390b1f10-7ba2-4d3a-142d-08d8149908a8)
## From https://github.com/maestro-auth-test/maestro-test1
- **Subscription**: 390b1f10-7ba2-4d3a-142d-08d8149908a8
- **Build**: 256776232
- **Date Produced**: 6/19/2020 9:37 PM
- **Commit**: 1290980782
- **Branch**: master
- **Updates**:
  - **Foo**: from  to 1.1.0
  - **Bar**: from  to 2.1.0

[marker]: <> (End:390b1f10-7ba2-4d3a-142d-08d8149908a8)

we dont want that in commit msg, kinda looks ugly so removed it

Reply:
you don't want the updated packages to be bold? I think thats important.

Reply:
We don't really care about that in commit message. Like the previous commit messages nothing was bold. https://github.com/dotnet/arcade-validation/commit/e788b97

Reply:
i guess its fine then.